packages/grid/src/elements/splitter/Pane.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState, useMemo, HTMLAttributes, forwardRef } from 'react';
+import { Splitter } from './components/Splitter';
+import { Content } from './components/Content';
+import { StyledPane } from '../../styled';
+import { PaneContext } from '../../utils/usePaneContext';
+
+const PaneComponent = forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>(
+  ({ children }, ref) => {
```

Comment:
```suggestion
  ({ children, ...props }, ref) => {
```

And spread the `{...props}` on the `<StyledPane>` below.

Reply:
Fixed in https://github.com/zendeskgarden/react-components/commit/d0dfbca241ee251c287613b4e9d38d9e8fa72bd9